sound/soc/sof/ipc4-topology.c

```diff
@@ -394,6 +394,7 @@ static int sof_ipc4_widget_setup_pcm(struct snd_sof_widget *swidget)
 	kfree(available_fmt->dma_buffer_size);
 free_copier:
 	kfree(ipc4_copier);
+	swidget->private = NULL;
```

Comment:
@libinyang should we add a Fixes: tag for this patch?

Reply:
@libinyang, how this patch is going to fix all four of those commits? It ought to conflict on top of any of them, no?
True, that this is going to be picked on a branch where these have been already present..